<?xml version="1.0"?>
<launch> 
	<arg name="rate" default="30"/>

   <node name="big_obj_sim_sensor" pkg="mdb_baxter_detection" type="obj_sim_sensor.py" required="true" output="screen">
	   <param name="model_name" type="str" value="exp_big_obj"/>
       <param name="radius" type="double" value="0.0675"/>
       <param name="obj_type" type="str" value="big_obj"/>
       <param name="rate" type="int" value="$(arg rate)"/>
   </node>

   <node name="small_obj_sim_sensor" pkg="mdb_baxter_detection" type="obj_sim_sensor.py" required="true" output="screen">
	   <param name="model_name" type="str" value="exp_small_obj"/>
       <param name="radius" type="double" value="0.03"/>
       <param name="obj_type" type="str" value="small_obj"/>
       <param name="rate" type="int" value="$(arg rate)"/>
   </node>

   <node name="box_sim_sensor" pkg="mdb_baxter_detection" type="obj_sim_sensor.py" required="true" output="screen">
	   <param name="model_name" type="str" value="exp_box"/>
       <param name="radius" type="double" value="0.115"/>
       <param name="obj_type" type="str" value="box"/>
       <param name="rate" type="int" value="$(arg rate)"/>
   </node>

</launch>


